.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: "CI"
+on:
+  pull_request:
+    paths-ignore:
+    - '**.md'
+    - '**.org'
+    - '.dir-locals.el'

```

Comment:
Should this exclude `.elpaignore` and all of `doc/**` as well?

Is there a way to avoid repeating the same `paths-ignore` for both `pull_request` and `push`?

Reply:
I don't know a way to avoid repeating the same pattern. But I guess that's a nit right?

Reply:
> But I guess that's a nit right?

Yes.